.github/workflows/create_translated_project.yaml

```diff
@@ -15,10 +15,22 @@ jobs:
       - name: Pull translations and build project
         run: |
           export QGEP_SOURCE_DIR=$(pwd)
-          pip install transifex-client
+          curl -OL https://github.com/transifex/cli/releases/download/v1.3.1/tx-linux-amd64.tar.gz
+          tar -xvzf tx-linux-amd64.tar.gz
           ./scripts/pull_ts_and_create_project.sh
       - uses: actions/upload-artifact@v2
         with:
           name: project
           path: project/
 
+
+# - name: tx sync (push -> pull)
+#   if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }}
+#   env:
+#     TX_TOKEN: ${{ secrets.TX_TOKEN }}
+#   run: |
+#     curl -OL https://github.com/transifex/cli/releases/download/v1.3.1/tx-linux-amd64.tar.gz
+#     tar -xvzf tx-linux-amd64.tar.gz
+#     ./tx push
+#     ./tx pull
+#     ./tx status
```

Comment:
@why-not-try-calmer what is the purpose of these commented lines?

Reply:
Thanks, it was a neglected leftover